attest/tpm.go

```diff
@@ -46,28 +45,26 @@ var (
 	aikTemplate = tpm2.Public{
 		Type:       tpm2.AlgRSA,
 		NameAlg:    tpm2.AlgSHA256,
-		Attributes: tpm2.FlagSignerDefault | tpm2.FlagNoDA,
+		Attributes: tpm2.FlagSignerDefault,
 		RSAParameters: &tpm2.RSAParams{
 			Sign: &tpm2.SigScheme{
 				Alg:  tpm2.AlgRSASSA,
 				Hash: tpm2.AlgSHA256,
 			},
-			KeyBits: 2048,
-			Modulus: big.NewInt(0),
+			KeyBits:    2048,
```

Comment:
Having a 256-byte random `ModulusRaw` might be a good idea here, it will prevent two applications from inadvertently creating the same AIK. If you don't care about that, leaving `ModulusRaw` empty is the right choice (and is what `tpm2-tools` does.

Reply:
SGTM about leaving it empty.